stubs/init-laravel/phpunit.xml.stub

```diff
@@ -4,28 +4,29 @@
          bootstrap="vendor/autoload.php"
          colors="true"
 >
-    <testsuites>
-        <testsuite name="Unit">
-            <directory suffix="Test.php">./tests/Unit</directory>
-        </testsuite>
-        <testsuite name="Feature">
-            <directory suffix="Test.php">./tests/Feature</directory>
-        </testsuite>
-    </testsuites>
-    <coverage>
-        <include>
-            <directory suffix=".php">./app</directory>
-        </include>
-    </coverage>
-    <php>
-        <env name="APP_ENV" value="testing"/>
-        <env name="BCRYPT_ROUNDS" value="4"/>
-        <env name="CACHE_DRIVER" value="array"/>
-        <!-- <env name="DB_CONNECTION" value="sqlite"/> -->
-        <!-- <env name="DB_DATABASE" value=":memory:"/> -->
-        <env name="MAIL_MAILER" value="array"/>
-        <env name="QUEUE_CONNECTION" value="sync"/>
-        <env name="SESSION_DRIVER" value="array"/>
-        <env name="TELESCOPE_ENABLED" value="false"/>
-    </php>
+  	<testsuites>
+    	<testsuite name="Unit">
+      		<directory suffix="Test.php">./tests/Unit</directory>
+    	</testsuite>
+    	<testsuite name="Feature">
+      		<directory suffix="Test.php">./tests/Feature</directory>
+    	</testsuite>
+  	</testsuites>
+  	<coverage/>
```

Comment:
What's this coverage?

Reply:
This is made when using `--migrate-configuration`. More details on the process : 

- create a new Laravel project [ composer create-project laravel/laravel test ]
- `composer require pestphp/pest --dev --with-all-dependencies`
- delete `phpunit.xml` file.
- `vendor/bin/pest --init`.
- `vendor/bin/pest` will display the Warning
- `vendor/bin/pest --migrate-configuration` will then create the phpunit.xml file you see in this PR, with that self closing `<coverage/>` tag.

The default new Laravel project `phpunit.xml` file has as `source` : 

```
<source>
    <include>
        <directory>app</directory>
    </include>
</source>
```

without any self closing `<coverage/>` tag. It seems to be more accurate. Do you want me to set these above lines instead of the actual ones in this PR?

Reply:
Yeah, I think it's best to just remove the coverage tag from both. It doesn't add anything unless there's something in it. 👍🏻

Reply:
Ok I removed them.